pkg/cmd/server/api/v1/conversions.go

```diff
@@ -307,6 +307,23 @@ func addConversionFuncs(scheme *runtime.Scheme) {
 			out.Location = in.Location
 			return nil
 		},
+		func(in *MasterVolumeConfig, out *internal.MasterVolumeConfig, s conversion.Scope) error {
+			if in.DynamicProvisioningEnabled != nil && !*in.DynamicProvisioningEnabled {
+				out.DynamicProvisioningEnabled = false
+				return nil
+			}
+			out.DynamicProvisioningEnabled = true
+			return nil
+		},
+		func(in *internal.MasterVolumeConfig, out *MasterVolumeConfig, s conversion.Scope) error {
```

Comment:
Added.  Looks like I just had to add it to the config in `BuildSerializeableMasterConfig`?  Pretty much exactly what I did for this in the types_test.